bpo-43753: Add Py_Is() and Py_IsNone() functions (#25227)
python/cpython: Misc/NEWS.d/next/C API/2021-04-06-20-52-44.bpo-43753.xUsHp1.rst

> Add :c:func:`Py_Is(x, y) <Py_Is>` function testing if the *x* object is the *y*
> object, same than ``x is y`` in Python. Add also :c:func:`Py_IsNone`,
> :c:func:`Py_IsTrue`, :c:func:`Py_IsFalse` functions to test if an object is,
> respectively, the ``None`` singleton, the ``True`` singleton or the ``False``
> singleton.
> Patch by Victor Stinner.

Reviewer: ```suggestion
Add the :c:func:`Py_Is(x, y) <Py_Is>` function to test if the *x* object is the
*y* object (same as ``x is y`` in Python). Add also the :c:func:`Py_IsNone`,
:c:func:`Py_IsTrue`, :c:func:`Py_IsFalse` functions to test if an object is,
respectively, the ``None`` singleton, the ``True`` singleton or the ``False``
singleton.
Patch by Victor Stinner.
```
